gno.land/cmd/gnoland/start.go

```diff
@@ -200,6 +202,13 @@ func execStart(c *startCfg, io commands.IO) error {
 		loadCfgErr error
 	)
 
+	// Attempt to initialize telemetry. If the enviroment variables required to initialize
+	// telemetry are not set, then the initialization will do nothing.
+	ctx := context.Background()
```

Comment:
Nitpick: you can inline this

Disturbingly, we don't utilize the command context in the node start method as a top-level controller, but defer to signal trapping :)

Reply:
I just removed it altogether as it currently serves no real purpose if we're never setting any deadline.